specification/commerce/resource-manager/Microsoft.Commerce/preview/2015-06-01-preview/commerce.json

```diff
@@ -27,6 +27,16 @@
         "externalDocs": {
           "url": "https://docs.microsoft.com/rest/api/commerce/usageaggregates"
         },
+        "x-ms-examples": {
+          "GetUsageAggregatesList": {
+            "$ref": "./examples/GetUsageAggregatesList.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location",
+          "final-state-schema": "#/definitions/UsageAggregationListResult"
+        },
```

Comment:
Actually, this is not a long running operation, and it should not be marked as LRO, so please remove this .

Reply:
yes, please do remove x-ms-long-running-operation

Reply:
I removed the long running operation attribute.

Just for background: I originally added this because the linter told me that this was required for the 202. Removing this might generate another linter flag.